Error when trying to load any map

The web demo appears to be broken at the moment. If you load in any map you get:

```
evented.js:140 Error: layers[70].layout.text-anchor: expected one of [center, left, right, top, bottom, top-left, top-right, bottom-left, bottom-right], "top-center" found
```

--- a/src/ocad-to-mapbox-gl-style.js
+++ b/src/ocad-to-mapbox-gl-style.js
@@ -281,7 +281,7 @@ const textLayer = (id, source, sourceLayer, scaleFactor, filter, element, colors
       'text-ignore-placement': true,
       'text-max-width': Infinity,
       'text-justify': justify,
-      'text-anchor': `${anchor}${anchor !== 'center' ? `-${justify}` : ''}`,
+      'text-anchor': `${anchor}${justify !== 'center' ? `-${justify}` : ''}`,
       'text-pitch-alignment': 'map',
       'text-rotation-alignment': 'map'
     },

